packages/frontend-core/src/components/grid/controls/ColumnsSettingButton.svelte

```diff
@@ -14,36 +17,64 @@
 
   const toggleColumn = async (column, permission) => {
```

Comment:
Do we need to modify `anyHidden` above?

Reply:
I did not realise about this logic... I think this should go. It should be active ONLY if open, otherwise it looks like it's focused or similar. Thoughts, @aptkingston ?

https://github.com/Budibase/budibase/assets/15987277/9ccbdc97-3220-4997-806a-9710a2a6792e

Reply:
It was designed to look this way, but I’m not strongly opinionated about it. The same behaviour happens with filters too - it’s in an active state of any filters are applied. Mainly just designed to make it very obvious that you have settings configured that are hiding rows or columns, since it could be confusing if you don’t realise that.

Reply:
I think my preference is still show this as highlighted to make it clear that there has been some deviation from the table schema.